sim/dex.ts

```diff
@@ -1139,6 +1124,7 @@ export class ModdedDex {
 			}
 
 			// gender
+			const template = dexes['base'].getTemplate(set.species || set.name);
```

Comment:
If we just always encode gender (0-1 character...) we can remove the need for having to `getTemplate` at all

Reply:
We support random gender.

Reply:
> We support random gender.

This is actually an argument _for_ doing as suggested by @scheibo .

Reply:
Sure.

Reply:
```suggestion
```